index.html

```diff
@@ -2905,6 +2915,166 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+A Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that a <a>subject</a> controls, or a <a>holder</a>
+has been designated to use, one or more particular confirmation methods. This
+binds a specific <a>claim</a> about a <a>subject</a> to one or more
+confirmation methods. In this way, an <a>issuer</a> explicitly gives guidance
+around how a <a>verifier</a> can validate that the <a>holder</a> is bound to
+the <a>claims</a> presented in a <a>verifiable credential</a>.
+        </p>
+
+        <p class="note">
+A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
+credential</a> without validating the confirmation method, or to use a
+different mechanism to validate whether the <a>holder</a> is bound to the
+presented <a>claims</a>, if required. Such decision can prevent impact on the
+<a>verifier</a>'s liability or lack thereof if not specified by other means
+such as a <code>termsOfUse</code> policy.
```

Comment:
```suggestion
A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
credential</a> without requiring use of the confidence method, or use a
different mechanism to increase their confidence about whether, for
example, the <a>holder</a> is the same <a>entity</a> the issuer made
<a>claims</a> about in the <a>verifiable credential</a>. Such a decision
can prevent impact on the <a>verifier</a>'s liability or lack thereof if not
specified by other means such as a <code>termsOfUse</code> policy.
```

I didn't change the last sentence here but it is a bit hard to parse. That can be fixed up editorially later.

Reply:
Some rewording of both sentences...
```suggestion
A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
credential</a> without requiring use of the <a>issuer's</a> specified 
confidence method, or use an entirely different mechanism to increase their 
confidence about whether, for example, the <a>holder</a> is the same 
<a>entity</a> the issuer made <a>claims</a> about in the <a>verifiable 
credential</a>. Such a decision might impact the <a>verifier</a>'s 
liability or lack thereof if not specified by other means such as a 
<code>termsOfUse</code> policy.
```